### DISCUSSION TITLE
video.js in outlook email?

### DISCUSSION THREAD

**Discussion by boxitrue** [videojs/video.js#7679] [Q&A] [answered]
Hi,

I'm trying to figure out if it's already known if video.js can present an embed video player in an email sent from outlook.

All I know is that outlook sends html.

I'm just not sure if that's a thing or not.  Do people embed video.js in html formatted emails? Is that a regular use case and does it work well?

**Comment by gkatsev** [comment, marked as answer]
Unfortunately, most email clients do not support embedded media. https://www.caniemail.com/search/?s=video

What most folks do is include a thumbnail with a "play" button that when clicked take you to the video.
